torch/_inductor/utils.py

```diff
@@ -1126,7 +1126,8 @@ def use_ck_template(layout, m, n, k):
         native_arch.split(":")[0]: native_arch
     }
     requested_supported_archs = [
-        requested_archs[k] for k in requested_archs.keys() & config.rocm.supported_arch
+        requested_archs[k]
+        for k in requested_archs.keys() & config.rocm.ck_supported_arch
```

Comment:
I believe lintrunner won't let me get away with the old formatting. Can try reverting and re-running the CI though

Reply:
What changed is `supported_arch` -> `ck_supported_arch` which got changed in order to avoid leaving impression that rocm pytorch works only on MI300; this is the ck inductor backend specific flag

Reply:
I see. Thanks!